app.js

```diff
@@ -196,4 +196,22 @@ form.addEventListener('submit', (e) => {
     e.preventDefault();
     word.textContent = 'The email must be in lowercase';
   }
-});
\ No newline at end of file
+});
+
+const objectStrg = {
+  name: '',
+  mail: '',
+  msj: '',
+};
+
+form.addEventListener('click', () => {
+  objectStrg.name = document.getElementById('name').value;
+  objectStrg.mail = document.getElementById('mail').value;
+  objectStrg.msj = document.getElementById('msg').value;
+  localStorage.setItem('stored', JSON.stringify(objectStrg));
+});
+
+const storedInfo = JSON.parse(localStorage.getItem('stored'));
+document.getElementById('name').value = storedInfo.name;
+document.getElementById('mail').value = storedInfo.mail;
+document.getElementById('msg').value = storedInfo.msj;
```

Comment:
- [ ] I suggest that you check if the data exist in the localStorage before setting the values to the input fields when the page loads. An example of how this can be achieved is below:
```
window.onload = () => {
  let savedFormData = localStorage.getItem('contactFormInput');
  savedFormData = JSON.parse(savedFormData);
  // Check if the form data object is found on localStorage
  if (savedFormData) {
  // populate inputs values if data was found
  // ex: nameInput.value = savedFormData.name
  }
};
```